torch/distributed/utils.py

```diff
@@ -327,3 +333,7 @@ def _replace_by_prefix(
         new_key = new_prefix + key[len(old_prefix) :]
         state_dict[new_key] = state_dict[key]
         del state_dict[key]
+
+
+def _data_ptr_allocated(tensor: torch.Tensor) -> bool:
```

Comment:
Is this missing an allow_in_graph? Or is it not used?

Reply:
Doesn't need it for now - It will in the future, but this makes it easier for me to track where I will eventually need to add it.